### ISSUE TITLE
assert freed

### ISSUE THREAD

**Issue by bitwes** [bitwes/Gut#28]
add an assert that verifies that an object has been freed.  Right now the only way I know how to do that is:
```
func is_freed(obj):
	return str(obj) == '[Deleted Object]'
```
There might be a better way but this has worked so far.

**Comment by bitwes** [comment]
``` python
var wr = weakref(obj)
if(wr.get_ref()):
  # freed
else:
  # not freed
```

**Comment by bitwes** [comment]
Implemented, in next version after 6.8
